modules/storage-api/src/main/java/org/apache/ignite/internal/storage/index/IndexRowPrefix.java

```diff
@@ -23,7 +23,7 @@
 public interface IndexRowPrefix {
```

Comment:
I believe that we should use InternalTuple for prefixes as well, there's no point in having additional interface now

Reply:
Removed this class